src/clims/models/workflow.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+from django.db import models
+from sentry.db.models import (FlexibleForeignKey, sane_repr)
+
+
+class Workflow(models.Model):
+    """
+    Represents a Workflow in an external workflow engine.
+
+    The details of the workflow, including which subworkflows it may have, are defined in the
+    external system. The entry in the database is used to connect internal logs to the external
+    workflow, e.g. Assignments.
+    """
+    __core__ = True
+
+    BACKEND_CAMUNDA = 'camunda'
+
+    # The definition of the workflow in the external system
+    external_id = models.TextField()
+
+    # The full name of the class that defines the workflow, e.g. lab_awesome.plugin.workflows.a
+    name = models.TextField()
+
+    # The version of the workflow, incrementing integer
+    version = models.IntegerField()
+
+    # True if this is the latest version of this workflow definition (with this definition_key)
+    latest = models.BooleanField()
+
+    # The backend type used
+    backend = models.TextField()
+
+    # The plugin (down to the version) that registered the workflow
+    plugin_registration = FlexibleForeignKey('clims.PluginRegistration')
+
+    # organization = FlexibleForeignKey('sentry.Organization', related_name='workflows')
```

Comment:
Should workflows not be tied to organizations?

Reply:
I feel like we might need to revisit that question later. I hit some unanswered questions when it comes to relating plugins and organizations. When you install a plugin today, they are available to all organizations, because I wasn't sure how we would want to solve it from a user perspective - it's not as simple as pip install in that case. Since the workflows follow directly from the plugin, they don't have a connection to the workflow now, because plugins are on org level.